test/index.test.js

```diff
@@ -42,6 +42,10 @@ describe('Precaution workflow', () => {
     mockFiles['networking_binding.go'] = fs.readFileSync('test/fixtures/go/src/vulnerable_package/networking_binding.go', 'utf8')
     mockFiles['bad_test_file.go'] = fs.readFileSync('test/fixtures/go/src/vulnerable_package/bad_test_file.go', 'utf8')
     mockFiles['hello.go'] = fs.readFileSync('test/fixtures/go/src/safe/hello_world.go', 'utf8')
+    // I mock the "vulnerable.js" and "vulnerable.ts" files as empty strings
```

Comment:
No need to say "I", perhaps:
```// Mock the files as empty strings, we don't analyse their content anyway```

Reply:
Ohh... I don't know why I do that mistake so many times.
I will fix it.